extensions/datasource/common/src/main/java/io/quarkus/datasource/common/runtime/DatabaseKind.java

```diff
@@ -0,0 +1,113 @@
+package io.quarkus.datasource.common.runtime;
```

Comment:
Mildly curious, this class has to be runtime? vs deployment time?

Reply:
I want to be able to reference it everywhere. Typically it's used in the qualifier of the JDBC driver specific CDI beans.